sdk/keyvault/keyvault-keys/swagger/README.md

```diff
@@ -35,3 +35,16 @@ directive:
     transform: >
       $["x-ms-client-name"] = "authenticationTag"
 ```
+
+### Update swagger enum values for LifetimeActionsType to reflect what the service actually returns
```

Comment:
Calling out this change - I wanted to unblock myself but we do have a thread going to make the enum values lowercase if possible, or at least update the x-ms-enum swagger to reflect what the service actually returns